tests/Stecman/Component/Symfony/Console/BashCompletion/CompletionHandlerTest.php

```diff
@@ -148,6 +173,10 @@ protected function createHandler($commandLine, $cursorIndex = null)
      */
     protected function getTerms($handlerOutput)
     {
+        if (null === $handlerOutput) {
```

Comment:
I've missed that implode statement with `\n`. Then sure we need to return a string here.

I'll send another PR about that then.